run_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script runs the SYCL-BLAS tests using the provided Dockerfile.
+# The intention is to provide a seamless alternative to .travis.yml, so that developers can locally test changes in a (somewhat) platform-agnostic manner without the usual delay that happens with travis.
+   - IMPL=COMPUTECPP CXX_COMPILER=clang++-6.0 CC_COMPILER=clang-6.0 TARGET=opencl
```

Comment:
I don't see any issue using clang-7.0. unless there is a particular reason, feel free to switch to clang-7.0

Reply:
Clang 7 isn't available in ubuntu xenial (16.04), so we can only support clang 6. If we want to switch our travis to clang 7, we'd also have to upgrade to 18.04 - which we might want to do, but I think it's beyond the scope of this PR.